client/src/r_draw.cpp

```diff
@@ -218,31 +218,61 @@ static void R_BuildFontTranslation(int color_num, argb_t start_color, argb_t end
 {
 	const palindex_t start_index = 0xB0;
 	const palindex_t end_index = 0xBF;
+	const palindex_t chexstart_index = 0x70;
+	const palindex_t chexend_index = 0x7F;
```

Comment:
Is this the kind of information that can be tied to the gamemode configurations?

Reply:
Possibly. I didn't look into any part of the code related to that other than to see how the current IWAD is determined. I changed only the one file to keep it as simple as possible. If there are other files you want to outsource some of the alterations/checks to in addition to r_draw.cpp, or even instead of, I'll look into seeing if I can do it.